platform-api/che-core-api-account/pom.xml

```diff
@@ -39,15 +39,16 @@
             <artifactId>che-core-api-user</artifactId>
             <version>${project.version}</version>
         </dependency>
-
-
-
         <dependency>
             <groupId>org.eclipse.che.core</groupId>
             <artifactId>che-core-api-workspace</artifactId>
             <version>${project.version}</version>
         </dependency>
-
+        <dependency>
+            <groupId>org.modelmapper</groupId>
+            <artifactId>modelmapper</artifactId>
+            <version>0.7.5</version>
```

Comment:
The same [comment](https://github.com/codenvy/che-core/pull/307#discussion_r41075416)